.nix/toolchain.nix

```diff
@@ -0,0 +1,7 @@
+{ rust-bin }:
+
+rust-bin.stable.latest.default.override {
+  extensions = [ "rust-src" "rust-analyzer" ];
+  targets = [ "arm-unknown-linux-gnueabihf" ];
```

Comment:
this seems fishy. i'd expect the default one to at least contain the appropriate target for the system